chapters/refereecommands.adoc

```diff
@@ -17,8 +17,23 @@ When the halt command is issued, no robot is allowed to move or <<Ball Manipulat
 There is a grace period of 2 seconds for the robots to brake.
 
 .Usage
-The halt command allows the referee to interrupt the game immediately whenever an emergency occurs (for example when a robot gets out of contol). It is
-also used to recalibrate the vision software during a game if the vision expert considers it necessary and the referee agrees and for <<Robot Substitution, robot substitution>>. Additionally, the referee is free to issue the halt command at will.
+The halt command allows the referee to interrupt the game immediately whenever an emergency occurs (for example when a robot gets out of contol). It is also used for <<Robot Substitution, robot substitution>>. Additionally, the referee is free to issue the halt command at will.
+
+Regarding the vision system, the game will be immediately halted in case of catastrophic failures, which includes:
+
+. Camera failures;
+. SSL-Vision crash;
+. Network failures;
+. Anything that causes a significant drop in the frame rate of the data;
+
+The game is continued in case of:
+
+. The ball is detected at least every 0.2s while not being covered by some robot or shadow;
+. All robots on the field are detected at least every 0.5s;
```

Comment:
So if the ball disappears for more than 0.3s without being covered or a robot disappears for more than 0.5s, the game will be halted?

Reply:
Yes, that's correct. We will try to make it visible for the vision expert if such thing happens.

Reply:
I'd say, in this situation, the game **may** be halted, not **will** be halted. There will not be any automatic tool that halts the game automatically.

Reply:
@g3force I'm fine with that either way, I just think it should be worded more clearly in the rules itself. In one case I'd want to implement some indicator on my screen if the data gets worse than the threshold, in the other case there's no need for that.

Reply:
I've added what @g3force said, is it clear enough?

Reply:
fine by me.